pkg/controller/argocd/rolebinding.go

```diff
@@ -124,12 +124,6 @@ func (r *ReconcileArgoCD) reconcileRoleBinding(name string, rules []v1.PolicyRul
 		Name:     role.Name,
 	}
 
-	if name == applicationController {
-		if err := applyReconcilerHook(cr, roleBinding, ""); err != nil {
-			return err
-		}
-	}
-
```

Comment:
Why did we remove this?

Reply:
Earlier we were applying the reconciler hook for openshift specific deployments. This would bind the openshift `Cluster Role admin` to the application controller SA. now we don't need this, since we are creating a custom admin role binding the instance with that.